scripts/build-vercel-functions.sh

```diff
@@ -0,0 +1,29 @@
+#!/bin/bash
```

Comment:
We could use `set -e` and/or `set -o pipefail`  here, that'd prevent having to add `|| exit 1` everywhere.

Reply:
Awesome tip, I completely forgot about those, didn't do much bash lately!